src/docs/modules/appendix/pages/release-notes.adoc

```diff
@@ -17,13 +17,29 @@ SPDX-License-Identifier: MPL-2.0
 [[bamm-2.0.0]]
 === Release 2.0.0
 
-- Introduces Entity inheritance. The new *Abstract Entity* Meta Model Element and *extends* attribute may be used to
-define a hierarchy of Entities in an Aspect Model. The concept of extending an Entity replaces the existing concept of
-refining an Entity.
+* Introduces Entity inheritance. The new *Abstract Entity* Meta Model Element and *extends*
+attribute may be used to define a hierarchy of Entities in an Aspect Model. The concept of extending
+an Entity replaces the existing concept of refining an Entity. The `bamm:refines` keyword was
+therefore removed as its semantics are superseded by Entity inheritance.
+* The `bamm:name` attribute has been removed, as it encoded redundant information. The "short" name
+  of an element is the local name part of its URI, i.e., the part following the `#` symbol.
 * The meta model elements `Unit`, `QuantityKinds` and their corresponding attributes were moved from
```

Comment:
The meta model elements `Unit`, `QuantityKinds` and their corresponding attributes have been moved from `unit` to the `bamm` name space. The `unit` namespace is now solely reserved for the units of the xref:bamm-specification:appendix:unitcatalog.adoc#unit-catalog-units[unit catalog] itself.